bin/doppler-gestures.py

```diff
@@ -43,29 +43,24 @@ def tonePlayer(freq, sync):
     """
 
     p = pyaudio.PyAudio()
-
-    A = float(2**15 - 1)
-    CHUNK2 = ((CHUNK+RATE-1)/RATE)*RATE
-
     stream = p.open(format=pyaudio.paInt16,
                     channels=2,
                     rate=RATE,
-                    frames_per_buffer=CHUNK2,
+                    frames_per_buffer=RATE,
                     output=True,
                     input=False)
 
     stream.start_stream()
     sync.set()
-    s = 0
-    j = 0
+
+    A = float(2**15 - 1)
```

Comment:
This should be defined as a constant somewhere. I want to actually fix up this whole setup and maybe even make some generic wrapper for pyaudio for doing threaded "realish time" audio processing